src/components/Item/index.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+
+import classNames from 'classnames';
+import Markdown from 'markdown-to-jsx';
+import { getComponent } from '../../components-registry';
+
+import { mapStylesToClassNames as mapStyles } from '../../utils/map-styles-to-class-names';
+
+export default function Item(props) {
+    const sectionStyles = props.styles?.self || {};
+
+    const sectionBorderWidth = sectionStyles.borderWidth ? sectionStyles.borderWidth : 0;
+    return (
+        <div
+            className={classNames(
```

Comment:
missing elementId and classes. I commented on this in more detail on the FeaturedItemSection component